src/main/webapp/index.html

```diff
@@ -19,26 +19,72 @@
   <head>
     <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
-    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="css\main.css">
+    <link rel="stylesheet" href="css\index.css">
     <script src="/js/navigation-loader.js"></script>
   </head>
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
       <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/community.html">Community</a></li>
-        <li><a href="/stats.html">Stats</a></li>
-        <li><a href="/chart.html">Charts</a></li>
-        <li><a href="/favouriteplaces.html">Favourite Places</a></li>
-
+        <li><a href="/"><img src="images/home-logo.png" height="25px"></a></li>
+        <li><a href="/">PUBLIC FEED</a></li>
+        <li><a href="/community.html">COMMUNITY</a></li>
       </ul>
     </nav>
-    <h1>CodeU Starter Project</h1>
-    <p>This is the CodeU starter project. Click the links above to login and visit your page.
-    You can post messages on your page, and you can visit other user pages if you have
-    their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <div id="logo">
+      <img src="images/logo.png" width="100%">
+    </div>
+    <div id = "parallax">
+      <div id="slogan-container">
+        <div id="slogan">
+          <p class="p1">SHARE LOVE<p>
+          <p class="p2">TOGETHER WE CAN BE MIRACLES<p>
+        </div>
+      </div>
+    </div>
+    <div id="info-container">
+      <div id="general-text">
+        <p class="p4">WORLD CHARITIES</p>
+        <p class="p5">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla vitae dignissim nisl. Donec eget aliquam neque. Proin libero dolor, consequat eget nisl id, fringilla dapibus leo.</p> 
+      </div>
+      <div id="info-section">
+        <div class="info">
+          <div class="img-container">
+            <img class="info-item-image" src = "images/location-logo.png" width = "100%">
+          </div>
+          <div class="text-container">
+            <a href="/favouriteplaces.html">LOCATIONS</a>
+            <p class="p3">The point of using Lorem Ipsum is that it has a more-or-less normal.</p>
+          </div>
+        </div>
+        <div class="info">
+          <div class="img-container">
+            <img class="info-item-image" src = "images/charts-logo.jpg" width = "100%">
+          </div>
+          <div class="text-container">
+            <a href="/chart.html">CHARTS</a>
+            <p class="p3">The point of using Lorem Ipsum is that it has a more-or-less normal.</p>
```

Comment:
(i don't know what the correct description for this would be)

Reply:
I will fix this